Parse-Dashboard/index.js

```diff
@@ -5,114 +5,48 @@
  * This source code is licensed under the license found in the LICENSE file in
  * the root directory of this source tree.
  */
-// Command line tool for npm start
-"use strict"
+"use strict";
 const packageJson = require('package-json');
 const basicAuth = require('basic-auth');
 const path = require('path');
 const jsonFile = require('json-file-plus');
 const express = require('express');
 
-const program = require('commander');
-program.option('--appId [appId]', 'the app Id of the app you would like to manage.');
-program.option('--masterKey [masterKey]', 'the master key of the app you would like to manage.');
-program.option('--serverURL [serverURL]', 'the server url of the app you would like to manage.');
-program.option('--appName [appName]', 'the name of the app you would like to manage. Optional.');
-program.option('--config [config]', 'the path to the configuration file');
-program.option('--port [port]', 'the port to run parse-dashboard');
-program.option('--allowInsecureHTTP [allowInsecureHTTP]', 'set this flag when you are running the dashboard behind an HTTPS load balancer or proxy with early SSL termination.');
+class ParseDashboard {
```

Comment:
The code doesn't pass through babel, but Node.js supports some features of ECMAScript 6 without the need for babel compilation. (See: https://nodejs.org/en/docs/es6/)